code/datums/wounds/_wounds.dm

```diff
@@ -42,6 +42,8 @@
 	var/mob/living/carbon/victim = null
 	/// The bodypart we're parented to
 	var/obj/item/bodypart/limb = null
+	/// A phantom body zone, for situations where the wound is tied to a bodypart that doesn't exist on the victim, like dismemberment
+	var/fake_body_zone
```

Comment:
```suggestion
	var/phantom_body_zone = null
```
You can really not describe it any better than this. Can you update the variable name to match the very fitting description?